### PR TITLE
content-13

### PR DESCRIPTION
**Pull request by CenkerAydin**

### CODE REVIEW

File: editor-components/content/content13/content13.module.scss [hunk 1]
```diff
@@ -1,96 +1,377 @@
 .container {
+    height: 100%;
     width: 100%;
-    padding: 50px 0;
+    background-image: url("https://craftohtml.themezaa.com/images/vertical-center-line-bg.svg");
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.module.scss:4]: this should be a prop in JSX oart. also we use google storage url for image links

File: editor-components/content/content13/content13.module.scss [hunk 2]
```diff
@@ -1,96 +1,377 @@
...
+                    color: var(--text-color-white);
+                    text-align: center;
+                    font-weight: 800;
+                    text-transform: inherit;
+                    font-size: 18px;
+                    padding: 16px 34px;
+                    border: none;
+                    border-radius: 5px;
+                    cursor: pointer;
+                    margin-left: 10px; // İsteğe bağlı: buton ile diğer içerik arasındaki boşluk
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.module.scss:80]: delete redundant comment

File: editor-components/content/content13/content13.module.scss [hunk 3]
```diff
@@ -1,96 +1,377 @@
...
-                min-width: 100%;
-            }
-            .header {
-                width: 100%;
+            flex-direction: row;
+
+        }
+    }
+}
+@media only screen and (max-width: 1024px) and (min-width:768px) {
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.module.scss:183]: using only max-width for media queries is enough

File: editor-components/content/content13/content13.tsx [hunk 4]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
-    let serverimg1 = require("./server-1.png");
-    let serverimg2 = require("./server-2.png");
-    let serverimg3 = require("./server-3.png");
-
     this.addProp({
       type: "string",
-      key: "title",
-      displayer: "Title",
-      value: "Create a Landing Page In a Minute Not Weeks",
+      key: "about-title",
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:12]: use camelCase for all keys

File: editor-components/content/content13/content13.tsx [hunk 5]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
+      displayer: "about-title",
+      value: "About resorts",
     });
 
     this.addProp({
       type: "string",
-      key: "subtitle",
-      displayer: "Subtitle",
-      value: "Key Featured For Landing Page",
+      key: "maintitle",
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:19]: use camelCase

File: editor-components/content/content13/content13.tsx [hunk 6]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
+      displayer: "maintitle",
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:20]:
```suggestion
      displayer: "Main Title",
```

File: editor-components/content/content13/content13.tsx [hunk 7]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
+    })
+
     this.addProp({
-      type: "number",
-      key: "itemCount",
-      displayer: "Item count in a row",
-      value: 4,
+      type: "string",
+      key: "buttonText",
+      displayer: "Text of Button",
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:34]:
```suggestion
      displayer: "Button Text",
```

File: editor-components/content/content13/content13.tsx [hunk 8]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
+      key: "phone",
+      displayer: "PhoneNumber",
+      value: "1 800 222 000"
+    })
+
+    this.addProp({
+      type: "image",
+      key: "image1",
+      displayer: "image1",
+      value: "https://craftohtml.themezaa.com/images/demo-hotel-and-resort-home-01.jpg"
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:49]: use google storage urls for images

File: editor-components/content/content13/content13.tsx [hunk 9]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
+    })
+
+    this.addProp({
+      type: "image",
+      key: "image2",
+      displayer: "image2",
+      value: "https://craftohtml.themezaa.com/images/demo-hotel-and-resort-home-02.jpg"
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:56]: use google storage urls for images

File: editor-components/content/content13/content13.tsx [hunk 10]
```diff
@@ -2,187 +2,101 @@ import * as React from "react";
...
-              )
-            )}
+            </div>
+          </div>
+          <div className={this.decorateCSS("right-page")}>
+            <div className={this.decorateCSS("image-container")}>
+              <span className={this.decorateCSS("text")}> Started In</span>
+              <span className={this.decorateCSS("year")}>1995</span>
+              <img src={this.getPropValue("image1")} alt="" className={this.decorateCSS("image1")} />
+              <img src={this.getPropValue("image2")} alt="" className={this.decorateCSS("image2")} />
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:93]: image alt should be non-empty.

also this images do not seem right and overflows in desktop and mobile previews.

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
please handle changes

File: editor-components/content/content13/content13.tsx [hunk 11]
```diff
@@ -1,188 +1,127 @@
...
-                }}>
-                  <div key={`cnt-4-card-${index}`} className={this.decorateCSS("card")}>
-                    <i>
-                      <img alt="server" src={card.image} />
-                    </i>
-                    <h5 className={this.decorateCSS("card-title")}>{card.title}</h5>
-                    <p className={this.decorateCSS("card-description")}>{card.description}</p>
-                  </div>
+          <div className={this.decorateCSS("title-container")}>
+            <span className={this.decorateCSS("about-title")}>{this.getPropValue("aboutTitle")}</span>
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:96]: if prop is empty, do not render relevant elements. do this operation for all props

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
please handle changes

File: editor-components/content/content13/content13.tsx [hunk 12]
```diff
@@ -1,188 +1,135 @@
...
-                }}>
-                  <div key={`cnt-4-card-${index}`} className={this.decorateCSS("card")}>
-                    <i>
-                      <img alt="server" src={card.image} />
-                    </i>
-                    <h5 className={this.decorateCSS("card-title")}>{card.title}</h5>
-                    <p className={this.decorateCSS("card-description")}>{card.description}</p>
-                  </div>
+          <div className={this.decorateCSS("title-container")}>
+            { isAboutTitleExist && <span className={this.decorateCSS("about-title")}>{isAboutTitleExist}</span>}
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:104]: do this for all
```suggestion
            { isAboutTitleExist && <span className={this.decorateCSS("about-title")}>{this.getPropValue("aboutTitle")}</span>}
```

File: editor-components/content/content13/content13.tsx [hunk 13]
```diff
@@ -1,188 +1,135 @@
...
+            { isMainTitleExist && <h2 className={this.decorateCSS("maintitle")}>{isMainTitleExist}</h2>}
+           { isDescriptionExist && <p className={this.decorateCSS("description")}>{isDescriptionExist}</p> }
+            
+            <div className={this.decorateCSS("button-icon")} >
+              <button className={this.decorateCSS("about-button")}>
+               {isButtonTextExist && <div className={this.decorateCSS("buttonText")}>{isButtonTextExist}</div> }
+              </button>
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:111]:
```suggestion
 {isButtonTextExist &&
              <button className={this.decorateCSS("about-button")}>
               <div className={this.decorateCSS("buttonText")}>{this.getPropValue("buttonText")}</div>
              </button>
```

File: editor-components/content/content13/content13.tsx [hunk 14]
```diff
@@ -1,188 +1,135 @@
...
+            </div>
+          </div>
+          <div className={this.decorateCSS("right-page")}>
+            <div className={this.decorateCSS("image-container")}>
+              {isTextExist && <h1 className={this.decorateCSS("text")}>{isTextExist}</h1> }
+              {isYearExist && <h2 className={this.decorateCSS("year")}>{isYearExist}</h2>}
+              <img src={this.getPropValue("image1")} alt="Image 1" className={this.decorateCSS("image1")} />
+              <img src={this.getPropValue("image2")} alt="Image 2" className={this.decorateCSS("image2")} />
+            </div>
           </div>
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:128]:
```suggestion
{(this.getPropValue("image1") || this.getPropValue("image2")) &&
          <div className={this.decorateCSS("right-page")}>
            <div className={this.decorateCSS("image-container")}>
              {isTextExist && <h1 className={this.decorateCSS("text")}>{isTextExist}</h1> }
              {isYearExist && <h2 className={this.decorateCSS("year")}>{isYearExist}</h2>}
              <img src={this.getPropValue("image1")} alt="Image 1" className={this.decorateCSS("image1")} />
              <img src={this.getPropValue("image2")} alt="Image 2" className={this.decorateCSS("image2")} />
            </div>
          </div>
          }
```

File: editor-components/content/content13/content13.tsx [hunk 15]
```diff
@@ -1,188 +1,135 @@
...
+          <div className={this.decorateCSS("title-container")}>
+            { isAboutTitleExist && <span className={this.decorateCSS("about-title")}>{isAboutTitleExist}</span>}
+            { isMainTitleExist && <h2 className={this.decorateCSS("maintitle")}>{isMainTitleExist}</h2>}
+           { isDescriptionExist && <p className={this.decorateCSS("description")}>{isDescriptionExist}</p> }
+            
+            <div className={this.decorateCSS("button-icon")} >
+              <button className={this.decorateCSS("about-button")}>
+               {isButtonTextExist && <div className={this.decorateCSS("buttonText")}>{isButtonTextExist}</div> }
+              </button>
+              <div className={this.decorateCSS("phone-icon")}>
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:112]:
```suggestion
{(this.getPropValue("phoneIcon") || isPhoneExist) &&
              <div className={this.decorateCSS("phone-icon")}>
```

File: editor-components/content/content13/content13.tsx [hunk 16]
```diff
@@ -1,188 +1,135 @@
...
+              <ComposerIcon
+                    propsIcon={{ className: this.decorateCSS("icon") }}
+                    name={this.getPropValue("phoneIcon")}
+                  />  
+                {isPhoneExist && <h1 className={this.decorateCSS("phone")}>{isPhoneExist}</h1>}
```
Comment (dogusmiuw) [inline editor-components/content/content13/content13.tsx:117]:
```suggestion
                {isPhoneExist && <h1 className={this.decorateCSS("phone")}>{this.getPropValue("phone")}</h1>}
```

**Review by dogusmiuw — CHANGES_REQUESTED** [review]
please handle changes

**Review by dogusmiuw — APPROVED** [review]
waiting for ui review

**Review by Ayazelar — APPROVED** [review]

**Pull request merged**